include/functions.h

```diff
@@ -2724,10 +2724,10 @@ s32 func_8013E2D4(Actor* actor, s16 arg1, s32 arg2, s32 arg3);
 s32 func_8013E3B8(Actor* actor, s16 cutscenes[], s32 cutsceneArrayLen);
 // void func_8013E4B0(void);
 s32 func_8013E5CC(Vec3f* param_1, Vec3s* param_2, Vec3f* param_3, Vec3f* param_4, Vec3f* param_5, Vec3f* param_6);
-// void func_8013E640(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE2 param_5, UNK_TYPE4 param_6, UNK_TYPE4 param_7);
+Actor* func_8013E640(GlobalContext* globalCtx, Actor* arg1, Actor* arg2, u8 arg3, s16 arg4, z_sub_unkStruct_8013E640* arg5, func_8013E640_arg6 arg6);
```

Comment:
Based on how `func_8013E640` just passes arg5 into arg6, I think it is best just leave arg5 as a `void *`, which is what I did in the recently merged #408. Then the arg6 func should cast it to whatever it needs.